resources/util_test.go

```diff
@@ -236,3 +238,79 @@ func TestLowerStructFieldNameToFieldName1(t *testing.T) {
 		return
 	}
 }
+
+func TestUnknownGroup(t *testing.T) {
+	gid, err := GetGID("unknowngroup")
+	if err == nil {
+		t.Errorf("expected failure, but passed with: %d", gid)
+	}
+}
+
+func TestUnknownUser(t *testing.T) {
+	uid, err := GetUID("unknownuser")
+	if err == nil {
+		t.Errorf("expected failure, but passed with: %d", uid)
+	}
+}
+
+func TestCurrentUserGroupByName(t *testing.T) {
+	// get current user
+	userObj, err := user.Current()
+	if err != nil {
+		t.Errorf("error trying to lookup current user: %s", err.Error())
+	}
+
+	currentUID := userObj.Uid
+	currentGID := userObj.Gid
+
+	var uid int
+	var gid int
+
+	// now try to get the uid/gid via our API (via username and group name)
+	if uid, err = GetUID(userObj.Username); err != nil {
+		t.Errorf("error trying to lookup current user UID: %s", err.Error())
+	}
+
+	if strconv.Itoa(uid) != currentUID {
+		t.Errorf("uid didn't match current user's: %s vs %s", strconv.Itoa(uid), currentUID)
+	}
+
+	if gid, err = GetGID(userObj.Username); err != nil {
+		t.Errorf("error trying to lookup current user UID: %s", err.Error())
+	}
+
```

Comment:
Remove this nl too. Try to group the uid stuff together and the gid stuff together. In general, use whitespace/new-lines to seperate different ideas and group lines together according to the logic they perform. If that's not clear ping me or purpleidea to discuss. For example: Here's how I would format one of these functions https://gist.github.com/jonathangold/88a6af5558b268ec64337661e05a3014